src/utilities/strategy-helpers/deployment-helper.ts

```diff
@@ -129,16 +129,17 @@ async function checkManifestStability(kubectl: Kubectl, resources: Resource[]):
 }
 
 async function annotateAndLabelResources(files: string[], kubectl: Kubectl, resourceTypes: Resource[], allPods: any) {
+    const repositoryId = await getRepositoryId(TaskInputParameters.githubToken);

```

Comment:
The value here would be a Promise right?

Reply:
we are using async/await pattern. Since we are using await on a Promise, the return value is of the type number.

Reply:
No no, I said this because I noticed your return type to be `Promise<number>`